translations/en-AU.json

```diff
@@ -4,5 +4,6 @@
   "terraApplication.unsavedChangesPrompt.multiplePromptMessageIntro": "There are unsaved changes made to the following:",
   "terraApplication.unsavedChangesPrompt.multiplePromptMessageOutro": "Changes will be lost if you don't save them. How do you want to proceed?",
   "terraApplication.unsavedChangesPrompt.acceptButtonText": "Don't Save",
-  "terraApplication.unsavedChangesPrompt.rejectButtonText": "Continue Editing"
+  "terraApplication.unsavedChangesPrompt.rejectButtonText": "Continue Editing",
+  "terraApplication.errorBoundary.defaultErrorMessage": "We are currently unable to complete your request. Try again later, and contact your support team if this problem continues."
```

Comment:
I'm not sure these error boundary failures are the types of failures that will just work if you try again. Something massively went wrong in your application.